### PR TITLE
feat: add configurable ticker duration

### LINKED ISSUE #235: Make `tickerDuration` configurable
Currently, the batcher write client lacks support for a user-configurable [ticker duration](https://github.com/parca-dev/parca-agent/blob/a9cc56cf0a58d846e7d2da717a0a1948c2cbf9a3/cmd/parca-agent/main.go#L138). Add a flag to enable users to set the duration as per their requirements.

### PR DESCRIPTION
**Pull request by me-diru**
- fixes #235
- added a flag and updated README

Signed-off-by: Rohit Dandamudi <rohitdandamudi.1100@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,6 +76,8 @@ Flags:
                                   with store.
       --insecure                  Send gRPC requests via plaintext instead of
                                   TLS.
+      --ticker-duration           Duration between batcher client writes.
```
Comment (brancz) [inline README.md:79]: Maybe `--batch-group-interval`? It's also always good to say in the help text what the default value is.

Comment (kakkoyun) [reply README.md:79]: I think considering the lingo we used already, I think `--batch-write-interval` would be a better fit.

**Comment by kakkoyun** [comment]
@me-diru I think after the changes you need to run `make README.md` and `make fmt`

**Review by kakkoyun — APPROVED** [review]
LGTM. Awesome job :tada:

**Pull request merged**